src/connect/component.jsx

```diff
@@ -1,5 +1,5 @@
 /* @flow */
-import { loadAxo } from "@paypal/connect-loader-component";
+import { loadAxo } from "@paypal/accelerated-checkout-loader/dist/loader.esm";
```

Comment:
Can we make the loader a top level export from this package? Bit worried that importing nested dependencies can create issues with future refactorings in this component.